### DISCUSSION TITLE
Topic probability distribution for zero-shot modeling ?

### DISCUSSION THREAD

**Discussion by YvesJacquot** [MaartenGr/BERTopic#2096] [Q&A]
Hi
Topic distribution is really great to get multiple topics for each document. However I am able to make it work only for clustered topics. Is there a way to get the topic distribution for zero-shot topics?
Thanks

**Comment by MaartenGr** [comment]
You can use `.transform` for that as it should give you back the cosine similarities between topic and document embeddings. You can then use any softmax function to transform them to "proper" probabilities.
